src/finder.js

```diff
@@ -32,7 +49,15 @@ export function isCommand (node) {
 }
 
 export function isNamespace (node) {
-  return type(node) === 'Object' && any(isCommand, values(node)) && node
+  return type(node) === 'Object' && any(v => isCommand(v) || isModule(v), values(node)) && node
```

Comment:
I feel that `v => isCommand(v) || isModule(v)` is *asking* to be *ramda'ed*, but I couldn't find a nice way to do it. 😁